src/helpers.js

```diff
@@ -1,3 +1,17 @@
+function isNulledBox(box) {
```

Comment:
Most of our helpers are used by ourselves, too. We cant just use another library. So why not make the public when they are there anyway? For the isNullerBox helper I agree that its not much code but it helps a lot to increase readability in the bbox function